### PR TITLE
Changing the path to the win sdk to x64

### PR DESCRIPTION
**Pull request by samcake**
- Also fix the bad lamda capturing of local variables

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -36,7 +36,7 @@ if (WIN32)
   if (MSVC10)
     set(WINDOW_SDK_PATH "C:\\Program Files\\Microsoft SDKs\\Windows\\v7.1 ")
   elseif (MSVC12)
-    set(WINDOW_SDK_PATH "C:\\Program Files (x86)\\Windows Kits\\8.1\\Lib\\winv6.3\\um\\x86 ")
+    set(WINDOW_SDK_PATH "C:\\Program Files (x86)\\Windows Kits\\8.1\\Lib\\winv6.3\\um\\x64 ")
```
Comment (birarda) [inline CMakeLists.txt]: we'll want CMake to check if it's a 64-bit or 32-bit generator and then set this accordingly

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8497931cb4b43eac49f1ab7b840235eff5d51a5d available at: https://gist.github.com/15a05ed4b32cd1075bcc<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6627-win64.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6627.dmg

File: CMakeLists.txt (hunk 1 above)
Comment (jherico) [inline CMakeLists.txt]: We need to test whether we're doing a 64 bit or 32 bit build and pick the right one, not just unilaterally switch to 64 bit hardcoded.

```
if ("${CMAKE_SIZEOF_VOID_P}" EQUAL "8")
  set(WINDOW_SDK_PATH "C:\\Program Files (x86)\\Windows Kits\\8.1\\Lib\\winv6.3\\um\\x64 ")    
else()
  set(WINDOW_SDK_PATH "C:\\Program Files (x86)\\Windows Kits\\8.1\\Lib\\winv6.3\\um\\x86 ")    
endif()
```

Comment (birarda) [inline CMakeLists.txt]: Thanks @jherico for adding the code with the comment too. Better man than I!

Comment (samcake) [inline CMakeLists.txt]: Can you help me with the corrrect syntax here ?

Comment (birarda) [inline CMakeLists.txt]:
```
if ("${CMAKE_SIZEOF_VOID_P}" EQUAL "8")
  set(WINDOW_SDK_FOLDER "x64 ")    
else()
  set(WINDOW_SDK_FOLDER "x86 ")    
endif()

set(WINDOW_SDK_PATH "C:\\Program Files (x86)\\Windows Kits\\8.1\\Lib\\winv6.3\\um\\${WINDOWS_SDK_FOLDER} ")    
```

**Comment by hifi-gustavo** [comment]
Build console output for commit: 94b6ca4450179b5b360b7765c96fc112799f0616 available at: https://gist.github.com/15a05ed4b32cd1075bcc<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6627-win64.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6627.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 93dbc92c613efd80a392355333c8f454d95b31d8 available at: https://gist.github.com/15a05ed4b32cd1075bcc<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/High+Fidelity-PR-6627-win64.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6627.dmg

**Pull request merged**